app/views/sessions/new.html.erb

```diff
@@ -8,6 +8,10 @@
         <div class="panel-body">
           <%= form_tag :login, html: { class: 'form-hozisontal' } do %>
             <div class="form-group">
+              <%= label_tag :name %>
```

Comment:
I see the need for adding a name input when creating a user, but it shouldn't be there when we authenticate a user.  The field can be blank and still work.  Keep it at email and password.